equinix-openapi-fabric/src/main/java/com/equinix/openapi/fabric/ApiClient.java

```diff
@@ -45,6 +45,7 @@
 import java.util.*;
 import java.util.Map.Entry;
 import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
```

Comment:
Is this an unused import? I don't see where it's used in this change.

Reply:
good point, removed